### ISSUE TITLE
Failure F23 is NOT a failure if you look at the Procedure

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#1543]
> Failure of 1.4.2 due to playing a sound longer than 3 seconds where there is no mechanism to turn it off

Agreed this is a failure, but the Procedure states:

> Check that there is a way in a Web page to turn off any sound that plays automatically for more than three seconds.

https://www.w3.org/WAI/WCAG22/Techniques/failures/F23

It does not have to be "in a Web Page" and should be something like "for the Web Page"

see also "if a mechanism is available" which may be options 'out of the Web Page' itself (OS, UA, AT)

> mechanism
> process or technique for achieving a result
> 
> NOTE
> The mechanism may be explicitly provided in the content, or may be relied upon to be provided by either the platform or by user agents, including assistive technologies.

**Comment by awkawk** [comment]
@jake-abma would you be satisfied with:

Check that there is a way @@ to turn off any sound that plays automatically @@in a Web page@@ for more than three seconds.

**Comment by jake-abma** [comment]
the problem is we get stuck at the same problem as mentioned in https://github.com/w3c/wcag/issues/1533

is a 'device setting' to turn of ALL sound not also @@ to turn off any sound that plays automatically @@in a Web page@@ ?

We need to talk specifically about the sound that plays due to something added to the Web Page, not sound in general

**Comment by bruce-usab** [comment]
> is a 'device setting' to turn of ALL sound not also ... 

It is explicitly not sufficient for satisfying the SC text.

Can this issue be resolved simply by updating the F23 procedure?

**Comment by patrickhlauke** [comment]
i'd say the normative SC wording should be patched first, otherwise the technique may appear to contradict the normative text, which is never a good thing... https://github.com/w3c/wcag/issues/1533

**Comment by bruce-usab** [comment]
I agree that the techniques appears to have some conflict with the normative text.

But, I think it is the technique which is misleading and that the normative text is sub-optimal but okay as-is.  Plus there seems to be a lot of resistance to even the most modest edits to the normative text.

**Comment by jake-abma** [comment]
> That is explicitly not sufficient for satisfying the SC text.
> 
> Can this issue be resolved simply by updating the F23 procedure?

Yes I think so, change...

> Procedure
> Check that there is a way in a Web page to turn off any sound that plays automatically for more than three seconds.

...to something like...

> Procedure
> Check that there is a way for the Web page only to turn off sound that plays automatically for more than three seconds.

**Comment by jake-abma** [comment]
Any comments for: (**Add to Survey...?**)

Yes I think so, change...

> Procedure
> Check that there is a way in a Web page to turn off any sound that plays automatically for more than three seconds.

...to something like...

> Procedure
> Check that there is a way for the Web page only to turn off sound that plays automatically for more than three seconds.

**Comment by patrickhlauke** [comment]
https://github.com/w3c/wcag/pull/1826 - but this depends on https://github.com/w3c/wcag/pull/1825 (or an equivalent PR that clarifies the need for pause/stop to work *independently* from other audio output)
